README.md

```diff
@@ -113,6 +113,52 @@ Run the following to set up the PV/PVC for RabbitMQ persistence
 `kubectl apply -f rabbitmq/haste-rabbitmq.yaml`
 
 ## Set up RabbitMQ with helm
+First, you have to create the user credentials for the rabbitmq server. These will be used by scripts to access the mq. The credentials are created using two secrets; one for the admin user creation, and another for all other users. Because of the way the rabbitmq container is initialized, we have to create the admin user twice, once in each secret. A way to get around this would be to modifiy the docker image, but we will probably want to keep using the official image.
+
+```
+# create admin secret
+kubectl create secret generic rabbitmq-admin-creds --from-literal=username=hasterabbit --from-literal=rabbitmq-password=pass0
+
+# create the other user secret, and additional config of vhost and permissions, that is otherwise lost when creating users this way.
+kubectl create secret generic rabbitmq-user-creds --from-literal=load_definition.json="{\
+   'users':[\
+      {\
+         'name':'hasterabbit',\
+         'password_hash':'$(python rabbitmq/rabbitmq_password_hasher.py pass0)',\
```

Comment:
issue here is we have the admin password in plaintext...

Reply:
@dahlo

Reply:
I guess its ok, they can just change it prior to deployment

Reply:
yes, that is mostly to show that you have to hash the password before putting it in the secret. Could change it to pick the password from a file instead.